pyproject.toml

```diff
@@ -1,12 +1,13 @@
 [tool.black]
 line-length = 119
 target_version = ['py37']
+ignore = ["C901"]
```

Comment:
Are you sure we need it here as well?

Reply:
ah true, thanks! I first thought it was `black` that triggers this error.